app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='form-container'>
+  <h2>Log in</h2>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %><br />
+      <%= f.password_field :password, autocomplete: "current-password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions">
+      <%= f.submit "Log in" %>
+    </div>
+  <% end %>
+
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] I can't login. The page doesn't submit.

![image](https://user-images.githubusercontent.com/51296741/152422118-1e1ff95d-35e8-462f-b5bd-7d5ae9113e84.png)
This may be due to `Routing Error` when I clicked on `Confirm my account`
![image](https://user-images.githubusercontent.com/51296741/152422435-f18063a3-ee54-4e39-bacd-e75084516858.png)
I will suggest you fix this.

Reply:
Hi @enaburekhan. The application works fine on my end and my partners. You can find below screenshots of a user confirming his email and logging in successfully

Reply:
![confirm email](https://user-images.githubusercontent.com/33420420/152439250-ee0bbfae-3108-43ed-891e-125d7a033fda.png)
![home](https://user-images.githubusercontent.com/33420420/152439254-f721f608-126a-4f21-8c31-c9289b7a28b3.png)
![login](https://user-images.githubusercontent.com/33420420/152439257-50611cb4-eef7-4479-b4f3-53b58c58c3fc.png)